src/_utils/_git_secret_tools.sh

```diff
@@ -630,7 +630,7 @@ function _decrypt {
   fi
 
   if [[ ! -z "$homedir" ]]; then
-    base="$base --homedir=$homedir"
+    base="$base --homedir \"$homedir\""
```

Comment:
Does it solve the whitespace issue? 
I have also tried different approaches with no luck.
But I did not try this one, I guess.

I have even tried to rewrite the whole function using array of arguments.

Reply:
I don't think it does solve the issue

Reply:
but yes, using arrays seems to be the answer